uni/lib/view/Widgets/schedule_row.dart

```diff
@@ -45,13 +54,37 @@ class ScheduleRow extends StatelessWidget {
                             mainAxisAlignment: MainAxisAlignment.center,
                             crossAxisAlignment: CrossAxisAlignment.center,
                             children: [
-                              ScheduleTimeInterval(begin: begin, end: end)
+                              ScheduleTimeInterval(
+                                  begin: widget.exam.begin,
+                                  end: widget.exam.end)
                             ]),
-                        ScheduleEventRectangle(subject: subject, type: type),
-                        IconButton(
-                            icon: const Icon(MdiIcons.calendarPlus, size: 30),
-                            onPressed: () =>
-                                Add2Calendar.addEvent2Cal(createExamEvent())),
+                        ScheduleEventRectangle(
+                            subject: widget.exam.subject,
+                            type: widget.exam.examType),
+                        Column(children: [
+                          IconButton(
+                              icon: const Icon(MdiIcons.calendarPlus, size: 30),
+                              onPressed: () =>
+                                  Add2Calendar.addEvent2Cal(createExamEvent())),
+                          if (!mainPage)
+                            IconButton(
+                                icon: clicked
+                                    ? const Icon(Icons.remove_red_eye_outlined,
+                                        size: 30)
+                                    : const Icon(Icons.remove_red_eye,
+                                        size: 30),
+                                tooltip: clicked
+                                    ? "Mostrar na Área Pessoal"
+                                    : "Ocultar da Área Pessoal",
+                                onPressed: () => setState(() {
+                                      clicked = !clicked;
+                                      clicked
+                                          ? hidden.add(widget.exam)
+                                          : hidden.remove(widget.exam);
+                                      AppSharedPreferences.saveHiddenExams(
```

Comment:
This could be a call to a new function i AppSharedPreferences which removes or adds a hiddenExam